File: module-java/src/main/java/com/zrq/test/point/demo/java/fragment/Fragment2.java
package com.zrq.test.point.demo.java.fragment;

import android.app.Fragment;
import android.graphics.Color;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.Nullable;

import com.zrq.test.point.annotation.TestEntryPoint;

/**
 * 描述：
 *
 * @author zhangrq
 * createTime 2020/12/7 16:50
 */
@TestEntryPoint("module-java-原生-Fragment")
public class Fragment2 extends Fragment {

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, Bundle savedInstanceState) {
        TextView textView = new TextView(container.getContext());
        textView.setText("module-java-原生-Fragment");
        textView.setTextSize(70);
        textView.setTextColor(Color.RED);
        return textView;
    }
}

File: module-kotlin/src/main/java/com/zrq/test/point/demo/kotlin/fragment/Fragment2.kt
package com.zrq.test.point.demo.kotlin.fragment

import android.app.Fragment
import android.graphics.Color
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import com.zrq.test.point.annotation.TestEntryPoint

/**
 * 描述：
 *
 * @author zhangrq
 * createTime 2020/12/7 16:50
 */
@TestEntryPoint("module-kotlin-原生-Fragment")
class Fragment2 : Fragment() {

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val textView = TextView(container!!.context)
        textView.text = "module-kotlin-原生-Fragment"
        textView.textSize = 70f
        textView.setTextColor(Color.RED)
        return textView
    }
}